src/plugins/demo-app/pages/BlueprintInteraction.tsx

```diff
@@ -1,19 +1,27 @@
-import { AuthContext, DmssAPI, JsonView } from '@development-framework/dm-core'
+import {
+  AuthContext,
+  DmssAPI,
+  JsonView,
+  NewEntityButton,
+  TReference,
+} from '@development-framework/dm-core'
 import React, { useContext, useEffect, useState } from 'react'
 import { SingleSelect } from '@equinor/eds-core-react'
 import { AxiosResponse } from 'axios'
 
 export const BlueprintInteraction = () => {
+  const dataSourceName = 'DemoApplicationDataSource'
```

Comment:
```suggestion
  const dataSourceName = 'DemoApplicationDataSource'
```
Do we want this to be hardcoded? Don't we have a JSON file containing this?

Reply:
I will create a separate issue for this, but I think it's a good idea!